src/Components/Question/Question.js

```diff
@@ -1,57 +1,39 @@
 import './Question.scss';
-// import { useParams, Navigate } from 'react-router-dom';
 
-const Question = ({ songs }) => {
+const Question = ({ songs, score, setScore, handleClick }) => {
   const year = Object.keys(songs)[0];
-  console.log(songs[year])
 
-  // need to figure out how to randomize answers
+  const shuffleSongs = () => {
+    let unshuffled = [
+      songs[year].song1,
+      songs[year].song2,
+      songs[year].song3,
+      songs[year].song4,
+      songs[year].song5
+    ];
+    
+    return unshuffled.sort(() => Math.random() - 0.5);
+  }
+
+  const songList =
+    shuffleSongs().map(song => {
+      return (
+        <div onClick={() => handleClick(song.id)} key={song.id}>
+          <img src={song.image_url} />
```

Comment:
Dont forget alt text for your images (if you're worried about 100% accessibility). Could just add the song.name or whatever other easy info you can grab